tunnel_endpoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+AGENT_TUNNEL_ENDPOINT=${AGENT_TUNNEL_ENDPOINT:-http://localhost:4040}
+TAILS_TUNNEL_ENDPOINT=${TAILS_TUNNEL_ENDPOINT:-http://localhost:4040}
+
+WAIT_INTERVAL=${WAIT_INTERVAL:-3}
+WAIT_ATTEMPTS=${WAIT_ATTEMPTS:-10}
+
+retrieve_endpoint () {
+        for _ in $(seq 1 "$WAIT_ATTEMPTS"); do
+                #TODO If wait attempts exceeded and we still haven't successfully gotten an endpoint, exit with status 1
```

Comment:
I think we should implement this TODO. In the branch where we do the waiting, we should check if current number of attempt is equal to max and then `exit 1` if it is.